task/buildah-oci-ta/0.1/buildah-oci-ta.yaml

```diff
@@ -267,7 +267,7 @@ spec:
       echo $container > /var/workdir/container_name
 
       # Save the SBOM produced by Cachi2 so it can be merged into the final SBOM later
-      if [ -d "/var/workdir/cachi2" ]; then
+      if [ -f "/var/workdir/cachi2/cachi2.env" ]; then
```

Comment:
If we are checking for files, should we not be checking for `output/bom.json` instead of `cachi2.env`?

Reply:
I saw the build fail due to a lack of /cachi2/cachi2.env in the dockerfile, so that was the file I checked for. Would it make any functional difference?

Reply:
Does `output/bom.json exists == cachi2.env exists`? I  *think* so, but why take the risk?

+1 to Andrew's suggestion, let's check the existence of the file we care about.